File: Assignments/cpp/basic concept oop/6.cpp
#include <iostream>
#include <string>
using namespace std;

class Person {
private:
    string name;
    int age;
    string country;

public:
    // Function to set the values of name, age, and country
    void setValues() {
        cout << "Enter your name: ";
        getline(cin, name);
        cout << "Enter your age: ";
        cin >> age;
        cin.ignore(); // To consume the newline left by cin
        cout << "Enter your country: ";
        getline(cin, country);
    }

    // Function to get and display the values of name, age, and country
    void displayValues() {
        cout << "Name: " << name << endl;
        cout << "Age: " << age << endl;
        cout << "Country: " << country << endl;
    }
};

int main() {
    Person person;

    // Set values from the user
    person.setValues();

    // Display the values
    person.displayValues();

    return 0;
}

File: 27Nov_2024_python/Asignments/advance python/6.py
class Person:
    def __init__(self, name, age):
        self.name = name  
        self.age = age    

    def display_info(self):
        print(f"Name: {self.name}, Age: {self.age}")

person1 = Person("John", 25)

print(f"Name: {person1.name}")
print(f"Age: {person1.age}")

person1.display_info()


